scripts/build.py

```diff
@@ -141,26 +162,32 @@ def get_cache_slug():
     CACHE_SLUG = get_cache_slug()
     BUILD_DIR = os.path.join('build', CACHE_SLUG)
 
+    # ensure_directory_exists trims file paths passed to it. Hence, directory
+    # paths require a trailing slash.
     # Process assets, copy it to build/[cache_slug]/assets
     ASSETS_SRC_DIR = os.path.join('assets', '')
     ASSETS_OUT_DIR = os.path.join(BUILD_DIR, 'assets', '')
     copy_files_source_to_target(ASSETS_SRC_DIR, ASSETS_OUT_DIR)
 
-    # Process third_party/generated/prod, copy it to
+    # Process third_party resources, copy it to
     # build/[cache_slug]/third_party/generated
-    THIRD_PARTY_GENERATED_SRC_DIR = os.path.join(
-        'third_party', 'generated', 'prod', '')
     THIRD_PARTY_GENERATED_OUT_DIR = os.path.join(
-        BUILD_DIR, 'third_party', 'generated', '')
-    copy_files_source_to_target(
-        THIRD_PARTY_GENERATED_SRC_DIR, THIRD_PARTY_GENERATED_OUT_DIR)
+        BUILD_DIR, 'third_party', 'generated')
+    process_third_party_libs(THIRD_PARTY_GENERATED_OUT_DIR)
 
     # Process extensions, copy it to build/[cache_slug]/extensions
     EXTENSIONS_SRC_DIR = os.path.join('extensions', '')
     EXTENSIONS_OUT_DIR = os.path.join(BUILD_DIR, 'extensions', '')
-    copy_files_source_to_target(EXTENSIONS_SRC_DIR, EXTENSIONS_OUT_DIR)
-
-    _build_files()
+    # Certain files' syntax become incorrect after minification and hence
+    # they are ignored.
+    EXTENSIONS_IGNORE = [os.path.join('extensions', 'interactions',
```

Comment:
Are these actually extensions, or path elements? I'm not sure the name here acutely describes the memory.